volumes/miovision/sql/create-table-volumes_15min.sql

```diff
@@ -3,66 +3,70 @@
 -- DROP TABLE miovision_api.volumes_15min;
 CREATE TABLE miovision_api.volumes_15min
 (
-  volume_15min_uid serial NOT NULL,
-  intersection_uid integer,
-  datetime_bin timestamp without time zone,
-  classification_uid integer,
-  leg text,
-  dir text,
-  volume numeric,
-  CONSTRAINT volumes_15min_pkey PRIMARY KEY (volume_15min_uid),
-  CONSTRAINT volumes_15min_intersection_uid_datetime_bin_classification__key UNIQUE (intersection_uid, datetime_bin, classification_uid, leg, dir)
+    volume_15min_uid serial NOT NULL,
+    intersection_uid integer,
+    datetime_bin timestamp without time zone,
+    classification_uid integer,
+    leg text,
+    dir text,
+    volume numeric,
+    CONSTRAINT volumes_15min_intersection_uid_datetime_bin_classification_pkey
+    PRIMARY KEY (intersection_uid, datetime_bin, classification_uid, leg, dir)
 )
+PARTITION BY RANGE (datetime_bin)
 WITH (
   OIDS=FALSE
 );
-ALTER TABLE miovision_api.volumes_15min
-  OWNER TO miovision_admins;
+
+ALTER TABLE miovision_api.volumes_15min OWNER TO miovision_admins;
 GRANT ALL ON TABLE miovision_api.volumes_15min TO rds_superuser WITH GRANT OPTION;
 GRANT ALL ON TABLE miovision_api.volumes_15min TO dbadmin;
+REVOKE ALL ON TABLE miovision_api.volumes_15min FROM bdit_humans;
 GRANT SELECT, REFERENCES, TRIGGER ON TABLE miovision_api.volumes_15min TO bdit_humans WITH GRANT OPTION;
+REVOKE ALL ON TABLE miovision_api.volumes_15min FROM covid_admins;
```

Comment:
I think we can get rid of this meow, cause all covid_admins are humans so they should be able to have select reference and trigger already